packages/contentful--app-scripts/src/utils.ts

```diff
@@ -1,31 +1,44 @@
-const fs = require('fs');
-const chalk = require('chalk');
-const inquirer = require('inquirer');
-const { cacheEnvVars } = require('../utils/cache-credential');
+import fs from 'fs';
+import chalk from 'chalk';
+import inquirer from 'inquirer';
+import { cacheEnvVars } from '../utils/cache-credential';
+import { Definition } from './definition-api';
+import { Organization } from './organization-api';
 
 const DEFAULT_MANIFEST_PATH = './contentful-app-manifest.json';
 
-const throwValidationException = (subject, message, details) => {
+export interface FunctionAppAction {
```

Comment:
Isn't that hosted app actions?

Reply:
No, this is slightly different and should be aligned with [schema](https://github.com/contentful/extensibility-api/blob/master/lib/entities/app-action/schema.ts), except `entryFile` field.